### ISSUE TITLE
make `-l, --locale <locale>` optional like `--help` says

### ISSUE THREAD

**Issue by jessefarinacci** [alexa/ask-cli#220]
## I'm submitting a...
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

On the `ask dialog` subcommand, it is listed that setting a locale is optional but it is not optional. Either make it actually optional or mark the `--help` output match reality. I prefer not to have to pass the `--locale` :)

## Expected Behavior

`ask dialog` should not require `--locale`

## Current Behavior

`ask dialog` fails with message `[Error]: Locale has not been specified.`

**CLI Snapshot**

```
$ ask dialog --help
Usage: ask dialog [options]

simulate your skill via an interactive dialog with Alexa

Options:
  -s, --skill-id <skill-id>  [OPTIONAL] The skill ID.
  -l, --locale <locale>      [OPTIONAL] Locale value for the resource
...

$ ask dialog
[Error]: Locale has not been specified.
```

## Steps to Reproduce (for bugs)

Run `ask dialog` without specifying a `--locale`

## Possible Solution

Please don't require `--locale` if possible.

## Your Environment and Context

* ask-cli version: 2.9.0
* Operating System and version: 19.4.0
* Node.js version used for development: v14.3.0
* NPM version used for development: 6.14.4

**Comment by kakhaUrigashvili** [comment]
@jessefarinacci opened PR https://github.com/alexa/ask-cli/pull/221

**Comment by habuma** [comment]
With ASK CLI v1, `--locale` was optional ONLY IF you were replaying a recorded session. Otherwise it was required. That's because the recorded session included the locale under which the session was recorded and that would be used.

I believe that with ASK CLI v2, the `--locale` parameter is semi-optional in the same way. Required on a new session, optional for replay of a recorded session. While not what you are asking for, it is at least consistent with ASK CLI v1.

That said, I looked at the PR and like the idea of making it entirely optional such that it uses the first locale from the manifest. That would be more convenient. But if you're going to do that, then I'd like that default locale to be explained in the help so that there are no surprises and so that the developer knows to put their locale of choice first in the manifest.

**Comment by RonWang** [comment]
Yes agree with @habuma that we need to clarify in the dialog command UX about which default locale gets used.

**Comment by kakhaUrigashvili** [comment]
fixed in 2.10.0
